### PR TITLE
added welcome page

### LINKED ISSUE #1: Deployment Environment
#### As a developer I want a continuous deployment environment setup for my new Rails application.

Acceptance criteria:
- [x] [GitHub](https://github.com/AppFolioOnboarding) is used for source
  control.
- [x] The project's version of both Ruby and Rails are used both locally and on
  Heroku.
- [x] The landing page, `/`, provides some content.
- [x] Every available controller action has corresponding tests.
- [x] [CircleCI](https://circleci.com/) is used for Continuous Integration.
- [x] Heroku is used to deploy the application.
- [x] Pull requests are automatically deployed as review applications.
- [x] The staging application automatically deploys whenever the master branch
  is updated.
- [x] The staging application automatically deploys only after all tests pass
  on CircleCI.
- [x] The staging application deployment can be promoted to the production
  application deployment.
- [x] [Pronto](https://github.com/mmozuras/pronto) is used to help with code
  style consistency.

Discussion Topics:
- [ ] Where can we discover how long it takes to return an HTTP response?
- [ ] When should we commit our code?

References:
* https://rvm.io/rvm/install#try-out-your-new-rvm-installation
* http://guides.rubyonrails.org/getting_started.html
* http://guides.rubyonrails.org/testing.html#functional-tests-for-your-controllers
* https://devcenter.heroku.com/articles/getting-started-with-rails4
* https://devcenter.heroku.com/articles/pipelines
* https://github.com/bbatsov/ruby-style-guide

### PR DESCRIPTION
**Pull request by lsq921224**
Closes #1

### CODE REVIEW

File: app/assets/javascripts/welcome.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (bboe) [inline app/assets/javascripts/welcome.js]: Do you need this file?

File: app/assets/stylesheets/welcome.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Welcome controller here.
```
Comment (bboe) [inline app/assets/stylesheets/welcome.scss]: Do you need this file?

File: app/helpers/welcome_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module WelcomeHelper
```
Comment (bboe) [inline app/helpers/welcome_helper.rb]: Do you need this file?

File: config/routes.rb [hunk 4]
```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  get 'welcome/index'
```
Comment (bboe) [inline config/routes.rb]: Is this route required?

File: test/controllers/welcome_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class WelcomeControllerTest < ActionDispatch::IntegrationTest
+  test 'should get index' do
```
Comment (bboe) [inline test/controllers/welcome_controller_test.rb]: Test looks great. You may need to change the path if the welcome-specific route is removed.

**Comment by bboe** [comment]
A few fixes to address. Great start!

**Comment by bboe** [comment]
This looks great. The last thing to do is to squash this into the three atomic commits that it is. I'll  help you if you haven't done that before.

**Comment by lsq921224** [comment]
I squashed this into the two commits, somehow the app.json in this branch does not have db set up in the first place.

**Comment by bboe** [comment]
Great work!

**Pull request merged**